docs/make-changes-to-brexit-checker-content.md

```diff
@@ -50,3 +50,13 @@ Change notes are defined [in a `change_notes.yaml` file](https://github.com/alph
 When a new change note has been deployed to production, you need to run a rake task to send the notification. The notification will be sent to all subscribers who would see this action on their results page.
 
 https://deploy.blue.production.govuk.digital/job/run-rake-task/parambuild/?TARGET_APPLICATION=finder-frontend&MACHINE_CLASS=calculators_frontend&RAKE_TASK=brexit_checker:change_notification[UUID]
+
+## Notifying a subset of subscribers
```

Comment:
I suggest we lower the level of this heading, since this section is really a sub-section of '## Adding change notes'

Reply:
Good point!

Reply:
Have amended